WL4EditorWindow.cpp

```diff
@@ -2003,3 +2004,33 @@ void WL4EditorWindow::on_actionNew_Room_triggered()
     // Clear everything in the new room
     ClearEverythingInRoom(true);
 }
+
+void WL4EditorWindow::on_actionEdit_Credits_triggered()
+{
+    // Set up parameters for the currently selected room, for the purpose of initializing the dialog's selections
+    DialogParams::CreditsEditParams *creditsEditParams =
+        new DialogParams::CreditsEditParams();
+
+    // Show the dialog
+    CreditsEditDialog dialog(this, creditsEditParams);
+    if (dialog.exec() == QDialog::Accepted)
+    {
+        /*int currentTilesetId = CurrentLevel->GetRooms()[selectedRoom]->GetTilesetID();
```

Comment:
Let's remove this commented code. Or better, remove the whole if-else clause because it appears to do nothing.

Reply:
(just keep `dialog.exec();` of course)